### PR TITLE
Security updates

### PR DESCRIPTION
**Pull request by jgonggrijp**
Please look specifically for code changes that look like they might be mistakes, or that look like they might cause trouble in some other way.

I'm doing the bug-hunting myself (already found one in #143).

### CODE REVIEW

File: recordings/admin.py [hunk 1]
```diff
@@ -24,12 +24,12 @@ class RecordingAdmin(admin.ModelAdmin):
         'public',
         ('dialect', admin.RelatedOnlyFieldListFilter),
         ('place', admin.RelatedOnlyFieldListFilter),
-        'place__country',
+        # 'place__country',
```
Comment (oktaal) [inline recordings/admin.py]: Remove?

File: recordings/conftest.py [hunk 2]
```diff
@@ -103,7 +105,9 @@ def places(countries_in_db):
...
 def recordings(dialects_in_db, places_in_db, audio_files):
     """ Return Recording instances which haven't been saved yet. """
-    return (
-        Recording(dialect=dialects_in_db[0], place=places_in_db[0], recording=File(audio_files[0])),
-        Recording(dialect=dialects_in_db[1], place=places_in_db[1], recording=File(audio_files[1])),
-    )
+    with File(audio_files[0]) as f0:
+        with File(audio_files[1]) as f1:
+            yield (
+                Recording(dialect=dialects_in_db[0], place=places_in_db[0], recording=f0),
```
Comment (oktaal) [inline recordings/conftest.py]: `with` and `yield` here could be problematic https://stackoverflow.com/questions/41881731/is-it-safe-to-combine-with-and-yield-in-python

File: recordings/signals_test.py [hunk 3]
```diff
@@ -19,20 +19,34 @@ def test_standardize_filename(dummy):
...
 @pytest.fixture
-def blank_web_safe(dialects_in_db, places_in_db, mp3_file):
+def django_mp3_file(mp3_file):
+    with File(mp3_file) as django_mp3_file:
+        django_mp3_file.name = op.basename(django_mp3_file.name)
+        yield django_mp3_file
+
+
+@pytest.fixture
+def django_amr_file(amr_file):
```
Comment (oktaal) [inline recordings/signals_test.py]: Same here with the `wield` and `yield`

File: recordings/admin.py (hunk 1 above)
Comment (jgonggrijp) [reply recordings/admin.py]: Good catch. I wanted to re-enable this later, but I forgot.

File: recordings/conftest.py (hunk 2 above)
Comment (jgonggrijp) [reply recordings/conftest.py]: This is OK, it is a [pytest idiom](https://docs.pytest.org/en/latest/fixture.html#fixture-finalization-executing-teardown-code).

**Pull request merged**